validator/flags/account.go

```diff
@@ -0,0 +1,143 @@
+	// for accounts which a user desires to delete from their wallet.
+	DeletePublicKeysFlag = &cli.StringFlag{
+		Name:  "delete-public-keys",
+		Usage: "Comma-separated list of public key hex strings to specify which validator accounts to delete",
+		Value: "",
+	}
+	// DisablePublicKeysFlag defines a comma-separated list of hex string public keys
+	// for accounts which a user desires to disable for their wallet.
+	DisablePublicKeysFlag = &cli.StringFlag{
+		Name:  "disable-public-keys",
+		Usage: "Comma-separated list of public key hex strings to specify which validator accounts to disable",
+		Value: "",
+	}
+	// EnablePublicKeysFlag defines a comma-separated list of hex string public keys
+	// for accounts which a user desires to enable for their wallet.
+	EnablePublicKeysFlag = &cli.StringFlag{
+		Name:  "enable-public-keys",
+		Usage: "Comma-separated list of public key hex strings to specify which validator accounts to enable",
+		Value: "",
+	}
+	// VoluntaryExitPublicKeysFlag defines a comma-separated list of hex string public keys
+	// for accounts on which a user wants to perform a voluntary exit.
+	VoluntaryExitPublicKeysFlag = &cli.StringFlag{
+		Name: "public-keys",
+		Usage: "Comma-separated list of public key hex strings to specify on which validator accounts to perform " +
+			"a voluntary exit",
+		Value: "",
+	}
+
+	// Backup flags.
+
+	// BackupDirFlag defines the path for the zip backup of the wallet will be created.
+	BackupDirFlag = &cli.StringFlag{
+		Name:  "backup-dir",
+		Usage: "Path to a directory where accounts will be backed up into a zip file",
+		Value: DefaultValidatorDir(),
+	}
+	// BackupPublicKeysFlag defines a comma-separated list of hex string public keys
+	// for accounts which a user desires to backup from their wallet.
+	BackupPublicKeysFlag = &cli.StringFlag{
+		Name:  "backup-public-keys",
+		Usage: "Comma-separated list of public key hex strings to specify which validator accounts to backup",
+		Value: "",
+	}
+	// BackupPasswordFile for encrypting accounts a user wishes to back up.
+	BackupPasswordFile = &cli.StringFlag{
+		Name:  "backup-password-file",
+		Usage: "Path to a plain-text, .txt file containing the desired password for your backed up accounts",
+		Value: "",
+	}
+
+	// Cosmetic flags.
+
+	// ShowDepositDataFlag for accounts.
+	ShowDepositDataFlag = &cli.BoolFlag{
+		Name:  "show-deposit-data",
+		Usage: "Display raw eth1 tx deposit data for validator accounts",
+		Value: false,
+	}
+	// ShowPrivateKeysFlag for accounts.
+	ShowPrivateKeysFlag = &cli.BoolFlag{
+		Name:  "show-private-keys",
+		Usage: "Display the private keys for validator accounts",
+		Value: false,
+	}
+	// SkipDepositConfirmationFlag skips the y/n confirmation prompt for sending a deposit to the deposit contract.
+	SkipDepositConfirmationFlag = &cli.BoolFlag{
+		Name:  "skip-deposit-confirmation",
+		Usage: "Skips the y/n confirmation prompt for sending a deposit to the deposit contract",
+		Value: false,
+	}
```

Comment:
What is this used for? Prysm doesn't send deposit to deposit contract?

This flag is only used here: https://github.com/prysmaticlabs/prysm/blob/f75a8efc0d6584233dd6f1e4c6d40d5493355c70/validator/accounts/wallet_create.go#L128

Please fix appropriately.